gui_app.py

```diff
@@ -531,6 +535,9 @@ def update_wine_check_progress(self, evt=None):
         self.gui.progress.config(mode='determinate')
         self.gui.progressvar.set(0)
         self.gui.winevar.set(self.gui.wine_dropdown['values'][0])
+        # FIXME: Probably need better test for failure condition:
+        if len(self.gui.winevar.get()) == 0:
+            self.gui.wine_check_button.state(['!disabled'])
```

Comment:
Could we add a status message if it fails or a tooltip?

Reply:
Perhaps something similar for the Logos release, too

Reply:
Yes, the Logos release check already has a status message. I'm adding one now to the wine binary check.